src/global/RuntimeParameters.h

```diff
@@ -69,6 +69,9 @@ inline auto& RuntimeParameters() {
         // which has the downside that the sibling optimization where VALUES are
         // dynamically pushed into `SERVICE` is no longer used.
         Bool<"cache-service-results">{false},
+        // Whether the results of a `LOAD URL` operation should be cached. The
+        // caching currently assumes the content of all URLs to be fixed.
```

Comment:
The second part of the comment is a bit confusing.
Do you mean "if this is set to true, then we expect that the contents of remote URLs that are loadad via a LOAD Update  do not change over time?